tools/__tasks__/compile/css/atoms.js

```diff
@@ -0,0 +1,58 @@
+const path = require('path');
+const execa = require('execa');
+const { target } = require('../../config').paths;
+
+const atomCssPrefix = 'atom';
+
+const fontMap = {
+    'f-serif-text': '\\"Guardian Text Egyptian Web\\", Georgia, serif',
+    'f-serif-headline': '\\"Guardian Egyptian Web\\", Georgia, serif',
+    'f-sans-serif-text':
+        '\\"Guardian Text Sans Web\\", \\"Helvetica Neue\\", Helvetica, Arial, \\"Lucida Grande\\", sans-serif',
+    'f-sans-serif-headline':
+        '\\"Guardian Sans Web\\", \\"Helvetica Neue\\", Helvetica, Arial, \\"Lucida Grande\\", sans-serif',
+};
+
+module.exports = {
+    description: 'Copy atom CSS to target',
+    task: () =>
+        execa
+            .shell(
+                `ls -d ${path.join(
+                    'node_modules',
+                    '@guardian',
+                    'atom-renderer',
+                    'dist'
+                )}/*/`
+            )
+            .then(dirs => {
+                const dirsArray = dirs.stdout
+                    .split('\n')
+                    .map(dir => dir.replace(/\/$/, ''));
+
+                return Promise.all(
+                    dirsArray.map(dir => {
+                        const dirName = dir.substr(dir.lastIndexOf('/') + 1);
+                        const dest = path.join(
+                            target,
+                            'stylesheets',
+                            `${atomCssPrefix}-${dirName}-article-index.css`
+                        );
+                        return execa('cp', [
+                            `${dir}/article/index.css`,
+                            dest,
+                        ]).then(() =>
+                            Promise.all(
+                                Object.entries(
+                                    fontMap
+                                ).map(([varName, varValue]) =>
+                                    execa.shell(
+                                        `perl -pe 's/var(--${varName})/${varValue}/g' ${dest}`
```

Comment:
is `perl` going to be installed on all machines which are likely to build this code?

Reply:
raaaaaaah 😭 nope. will find another way.

Reply:
@SiAdcock the difference between Mac OS/GNU `sed`s is problematic. My "fix" solved the problem in TC but it was not working on my local machine anymore. I'll do that in JS but would like to avoid reading/writing the contents of the file. Mariot mentioned a gulp package to do that; is this something I could add as a dependency?

Reply:
You probably won't like it, but I had to move the postcss stuff out of `compile-css`, into its own module. Disentangling SASS compilation and PostCSS was necessary for atoms styles to be processed just like any other. And it's working:
![screen shot 2017-11-13 at 15 21 43](https://user-images.githubusercontent.com/629976/32733269-b14faf8a-c886-11e7-9b8e-0cc2f1d79e31.png)
😅

Reply:
No way, I think it looks great! Sorry for all the fiddly-ness